apps/condo/domains/notification/adapters/redStore/redStoreNotificationSender.js

```diff
@@ -0,0 +1,69 @@
+const { get } = require('lodash')
+
+
+const { fetch } = require('@open-condo/keystone/fetch')
+
+const { APS_RESPONSE_STATUS_SUCCESS } = require('../apple/constants')
+
+
+class RedStoreNotificationSender {
+    _projectId = null
+    _serviceToken = null
+    _url = null
+
+    /**
+     * @param config
```

Comment:
If you like to provide some documentation using JSDoc - please provide config object expected structure and some description for these fields

Reply:
ok. rm it